From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Guilherme Amadio <amadio@cern.ch>
Date: Wed, 12 May 2021 15:42:57 +0200
Subject: [PATCH] Add missing include to simdarray.h

---
 Vc/common/simdarray.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Vc/common/simdarray.h b/Vc/common/simdarray.h
index 67d1c23..50f89cd 100644
--- a/Vc/common/simdarray.h
+++ b/Vc/common/simdarray.h
@@ -33,6 +33,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //#include "../IO"
 
 #include <array>
+#include <limits>
 
 #include "writemaskedvector.h"
 #include "simdarrayhelper.h"
